src/connector/client.js

```diff
@@ -35,6 +35,7 @@ if (!isTopWindow) return;
 Zotero.GoogleDocs = {
 	config: {
 		fieldURL: 'https://www.zotero.org/google-docs/?',
+		brokenFieldURL: 'https://www.zotero.org/google-docs/?broken=',
```

Comment:
Is there any reason to pass `broken` to the server? If not, we should use `#` instead.

Reply:
I should have used # for these from the beginning, there's no reason to be passing anything to the server. Would have to do a bunch of compatibility code to transition to hashes. We can do that if you think it's a privacy concern, but I don't think it is.

Reply:
It's OK.